inc/colour.h

```diff
@@ -33,24 +33,8 @@ typedef struct s_colour
 	float	b;
 }	t_colour;
 
-typedef enum e_col_name
```

Comment:
If e_col_name has been moved to a different file, this has to be done in the bonus file "colour_bonus.h" as well. Sames goes for any other changes as Macros, function declarations, etc.

Reply:
A very uncool message. We still need duplication :/

Reply:
What must be done, must be done

Reply:
True.